skiko/src/commonMain/kotlin/org/jetbrains/skia/CodePoint.kt

```diff
@@ -0,0 +1,51 @@
+package org.jetbrains.skia
+
+// Copied from CharHelpers.skiko.kt
+// TODO Remove once it's available in common stdlib https://youtrack.jetbrains.com/issue/KT-23251
+internal typealias CodePoint = Int
+
+// Copied from CharHelpers.skiko.kt
+/**
+ * The minimum value of a supplementary code point, `\u0x10000`.
+ */
+private const val MIN_SUPPLEMENTARY_CODE_POINT: Int = 0x10000
+
+// Copied from CharHelpers.skiko.kt
+/**
+ * Converts a surrogate pair to a unicode code point.
+ */
+internal fun toCodePoint(high: Char, low: Char): CodePoint =
+    (((high - Char.MIN_HIGH_SURROGATE) shl 10) or (low - Char.MIN_LOW_SURROGATE)) + MIN_SUPPLEMENTARY_CODE_POINT
+
+// Copied from CharHelpers.skiko.kt
+internal fun Int.charCount(): Int = if (this >= MIN_SUPPLEMENTARY_CODE_POINT) 2 else 1
+
+internal val String.codePointsAsIntArray: IntArray
+    get() = codePoints.toList().toIntArray()
+
+internal val String.codePoints
+    get() = codePointsAt(0)
+
+internal fun String.codePointsAt(index: Int) = sequence {
```

Comment:
The code in `CharHelpers.skiko.kt` mentioned is actually like:
```suggestion
internal val CharSequence.codePoints get() = sequence {
```

Reply:
Yes, the copy is after my changes in https://github.com/JetBrains/compose-multiplatform-core/pull/1869
This is more generic and is not worse, so why change it?

Reply:
`String`/`CharSequence` consistency

Reply:
We have it in more than one place; the one I copied it from uses `String`, but sure, `CharSequence` is better.